features/notifications-apps.yml

```diff
@@ -0,0 +1,19 @@
+name: Notifications from service workers and installed apps
+# This feature is for the platform-specific nature of notifications support
+# in Chrome Android and Safari iOS, and anchors to the Caniuse feature.
+#
+# See `features/notifications.yml` for the full Notifications feature, which
+# does not align with Caniuse.
+description: The Notifications API, when used from a service worker for Chrome Android or home screen app for Safari iOS, displays messages to the user, typically using the notification system of the host operating system.
+spec: https://notifications.spec.whatwg.org/
+caniuse: notifications
+status:
+  baseline: low
```

Comment:
Oh, I'm just realizing that we need a `baseline_low_date` field here.
I think it needs to be 2023-03-27, which is Safari iOS 16.4's date.

Reply:
(Creating an issue backlink here, no action)

Some day we should handle this from version numbers, see https://github.com/web-platform-dx/web-features/issues/2438